offixsdk/src/main/java/org/aerogear/offixsdk/Extension.kt

```diff
@@ -0,0 +1,48 @@
+package org.aerogear.offixsdk
+
+import android.util.Log
+import com.apollographql.apollo.ApolloCall
+import com.apollographql.apollo.api.Operation
+import org.aerogear.offixsdk.persistence.Mutation
+import org.json.JSONObject
+
+val application = Application
+
+/*  Extension function on ApolloCall which will be used by the user while making a call request.
+
+   @param mutation whose data will be stored in the database if network connection is not there.
+   @param callback Callback which will handle the response or a failure exception.
+ */
+
+fun ApolloCall<Any>.offQueue(
```

Comment:
Man! So we can extend apollo library without forking stuff? This is awesome.
I would love more info about it in the comment. This is the core engine